src/platforms/shared.rs

```diff
@@ -3,10 +3,8 @@
 //!
 
 /// The build options shared by all platforms.
-pub const SHARED_BUILD_OPTS: [&str; 18] = [
+pub const SHARED_BUILD_OPTS: [&str; 16] = [
     "-DPACKAGE_VENDOR='Matter Labs'",
-    "-DCLANG_VENDOR='Matter Labs'",
```

Comment:
Were these unnecessary?

Reply:
Yeah, deprecated. Forgot to put some comments in the PR.